pkg/operator/starter.go

```diff
@@ -0,0 +1,94 @@
+package operator
+
+import (
+	"context"
+	"os"
+	"time"
+
+	apiextclientv1 "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	"github.com/openshift/library-go/pkg/operator/loglevel"
+
+	operatorconfigclient "github.com/openshift/lws-operator/pkg/generated/clientset/versioned"
+	operatorclientinformers "github.com/openshift/lws-operator/pkg/generated/informers/externalversions"
+	"github.com/openshift/lws-operator/pkg/operator/operatorclient"
+)
+
+const (
+	podNamespaceEnv   = "POD_NAMESPACE"
+	operatorNamespace = "openshift-lws-operator"
+	workQueueKey      = "key"
+)
+
+func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error {
+	kubeClient, err := kubernetes.NewForConfig(cc.ProtoKubeConfig)
+	if err != nil {
+		return err
+	}
+
+	dynamicClient, err := dynamic.NewForConfig(cc.ProtoKubeConfig)
+	if err != nil {
+		return err
+	}
+
+	apiextensionClient, err := apiextclientv1.NewForConfig(cc.KubeConfig)
+	if err != nil {
+		return err
+	}
+
+	operatorConfigClient, err := operatorconfigclient.NewForConfig(cc.KubeConfig)
+	if err != nil {
+		return err
+	}
+	operatorConfigInformers := operatorclientinformers.NewSharedInformerFactory(operatorConfigClient, 10*time.Minute)
```

Comment:
we can use NewKubeInformersForNamespaces to watch all resources we need

Reply:
Agree. Updated.